Cargo.lock

```diff
@@ -32203,7 +32089,7 @@ dependencies = [
  "futures",
  "glob-match",
  "hex",
- "libp2p",
+ "libp2p 0.52.4",
```

Comment:
Just notice that `zombienet-orchestrator` still requires `libp2p 0.52.4`

Reply:
I've made a [PR](https://github.com/paritytech/zombienet-sdk/pull/285) for `zombienet-sdk`